build.sbt

```diff
@@ -1,32 +1,40 @@
-import sbtcrossproject.crossProject
+lazy val scalaVersionsJS  = Seq("2.13.4", "2.12.12", "2.11.12")
+lazy val scalaVersionsJVM = "3.0.0-M2" +: scalaVersionsJS
```

Comment:
Maybe we can keep 2.10.7 version for JVM at least? I'm try to avoid dropping backward compatibility without serious reasons (we all have our share of legacy projects).